src/move-node.js

```diff
@@ -1,13 +1,13 @@
 const utils = require('./utils');
 const moveShard = require('./move-shard');
 
-const moveNode = async (toNode) => {
+const replaceNodeInStandAloneCouch = async (toNode) => {
```

Comment:
Standalone is an interesting word choice. It made me chuckle. Gives CouchDb some sort of strength of will to stand alone.... 
Let's call this `replaceForSingleNode` and not give CouchDb the qualities required to stand alone :D